keps/sig-cluster-lifecycle/0015-kubeadm-join-control-plane.md

```diff
@@ -361,29 +362,40 @@ As of today kubeadm supports two solutions for storing cluster certificates:
 
 The proposed solution for case 1. "Cluster certificates stored on file system",
 requires the user/the higher level tools to execute an additional action _before_
-invoking `kubeadm join --master`.
+invoking `kubeadm join --control plane`.
 
 More specifically, in case of cluster with "cluster certificates stored on file
-system", before invoking `kubeadm join --master`, the user/higher level tools
-should copy control plane certificates from an existing master node, e.g. bootstrap master
+system", before invoking `kubeadm join --control plane`, the user/higher level tools
+should copy control plane certificates from an existing node, e.g. the bootstrap control plane
 
 > NB. kubeadm is limited to execute actions *only*
 in the machine where it is running, so it is not possible to copy automatically
 certificates from remote locations.
 
-Then, the `kubeadm join --master` flow will take care of checking certificates
+Then, the `kubeadm join --control plane` flow will take care of checking certificates
 existence and conformance.
 
 As stated above, supporting for Cluster certificates stored in secrets is a non goal
 for this proposal.
 
 #### `kubeadm upgrade` for HA clusters
 
-Nothing in this proposal prevents implementation of `kubeadm upgrade` for HA cluster.
+The `kubeadm upgrade` workflow as of today is composed by two high level phases, upgrading the
+control plane and upgrading nodes.
 
-Further detail will be provided in a subsequent release of this KEP when all the detail
+The above hig-level workflow will remain the same also in case of clusters with more than
+one control plane instances, but with a new sub-step to be executed on secondary control-plane instances:
+
+1. Upgrade the control plane
+
+  1. Run `kubeadm upgrade apply` on a first control plane instance [No changes to this step]
+  1. Run `kubeadm upgrade node experimental-control-plane` on secondary control-plane instances [new step]
```

Comment:
The `Run...` lists need an extra space to be rendered properly (nesting lists in Markdown is done with 3 spaces)